index.bs

```diff
@@ -749,6 +743,12 @@ XRStationaryReferenceSpace {#xrstationaryreferencespace-interface}
 An {{XRStationaryReferenceSpace}} represents a tracking space that the user is not expected to move around within. Tracking in a {{stationary}} reference space is optimized for the assumption that the user will not move much beyond their starting point, if at all. For devices with [=6DoF=] tracking, {{stationary}} reference spaces should emphasize keeping the origin stable relative to the user's environment.
 
 <pre class="idl">
+enum XRStationaryReferenceSpaceSubtype {
```

Comment:
Regarding #532:
This change moves the IDL closer to the definition it will link to. However, these values are most relevant in `XRSession`, and the definitions actually describe the behavior of values passed to `XRSession::requestReferenceSpace()` (as part of `XRReferenceSpaceOptions`). As noted in #532, it probably makes sense to move the definitions/prose and IDL (including other things like `XRReferenceSpaceOptions` to `XRSession`.